Item.cpp

```diff
@@ -6,30 +6,33 @@
 
 Item::Item(std::string st) {
 
-	// Splits string by words and assigns the stats from array
-	std::stringstream ssline(st);
-	std::string* stats = new std::string[5]{};
+	statNum = 5;
+
+	std::stringstream ssline(st);					// Convert stream into stringstream.
+	std::string* stats = new std::string[5]{};		
 
 	int i = 0;
 
-	while (ssline.good() && i < 5) {
+	// Checks if the stream raises any errors and extracts the words to be put into the array. 
+	while (ssline.good() && i < statNum) {
 		ssline >> stats[i];
 		++i;
 	}
 
-	type = stats[0];								// Specifies what class of item it is
-	itemID = std::stoi(stats[1]);					// Item name/more specific 
+	type = stats[0];								
+	itemID = std::stoi(stats[1]);					

```

Comment:
Sorry, I just thought of this now, I should've when you first made it; this is the Item class, so I think it's better if you just call this id and not itemID.